### PR TITLE
Improvement read from cur dir

### PR DESCRIPTION
**Pull request by comatory**

### CODE REVIEW

**Comment by comatory** [comment]
I'm adding an option to allow for getting unread count of mails. Since the example shows that you can point it to `~/mail/INBOX/new`, that's fine but I think most people leave their email clients open. But maybe you still want to have the unread count. I added a new flag to enable this. Basically it reads both `new` and `cur` directories but filters messages with *S* (seen) flag from `cur` directory.

Right now it's displayed as (x+y) where x is `cur` and y is `new` directory because I'm not good with shell and could not figure out how to sum these two numbers. So for now they're represented as two numbers but it'd be nicer to have them as one.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Tmux maildir counter
 
+This is a fork of [tmux-maildir-counter](https://github.com/tmux-plugins/tmux-maildir-counter) with some enhancement.
+
```
Comment (erickpintor) [inline README.md]: Could you remove this line? Everything else looks good.

File: README.md [hunk 2]
```diff
@@ -19,10 +19,22 @@ set -g @maildir_counters '~/Mail/Personal/INBOX/new|~/Mail/Work/INBOX/new'
 
 Then, add the `maildir_counter` to your left/right status bar where `N` is the index of the folder you want to show.
 
-```tmux.confi
+```tmux.config
```
Comment (imomaliev) [inline README.md]: This should be `tmux.conf`

**Comment by erickpintor** [comment]
Thank you for the update. I think you need to rebase, though.

**Comment by comatory** [comment]
> Thank you for the update. I think you need to rebase, though.

Rebased and force pushed.

**Comment by erickpintor** [comment]
Thanks a lot!

**Pull request merged**